pkg/types/openstack/cloudinfo.go

```diff
@@ -0,0 +1,14 @@
+package openstack
+
+// CloudInfoData is used to keep information about the underlying OpenStack cloud
+var CloudInfoData *CloudInfo
```

Comment:
I would strongly suggest against a global variable. There is already now AWS and Azure metadata like information stored in install config asset try to use that.